src/main/scala/zooklabs/enum/Trials.scala

```diff
@@ -29,16 +29,21 @@ object Trials {
     val value    = "lap"
     val ordering = Ascending
   }
+  case object Overall extends Trials {
+    val value    = "overall_league"
+    val ordering = Descending
+  }
```

Comment:
One issue with adding overall as part of the other trials is it will get a redundant update as part of leagueRepository.updateLeagues before then getting updated+reordered in updateOverallLeague

Reply:
Could just filter it out I suppose